src/mgmt/cli_mgmt.c

```diff
@@ -160,6 +160,16 @@ fill_sys_info(Mgmt__GetAttachInfoResp *resp, struct dc_mgmt_sys_info *info)
 		return -DER_INVAL;
 	}
 
+	if (strnlen(resp->sys, sizeof(info->system_name)) > 0) {
```

Comment:
If an invalid system name could be defined, it could be useful to test if the result of` strnlen()` is equal to` sizeof(info->system_name)` to detect it.

Reply:
Good idea, I'll do that.

Reply:
Actually it looks like the `copy_str` macro is covering that case.

Reply:
OK thanks for the info and sorry for the disturbance.